src_docs/manual/en/HowTo_SetUpTeamProject.xml

```diff
@@ -13,451 +13,449 @@
+    use are provided in the next section.</para>
+
+    <variablelist>
       <varlistentry>
-		<term>repository type</term>
-		<listitem>
+        <term>repository type</term>
+        <listitem>
           <para>This can be either <emphasis>http</emphasis> (which includes
           https), <emphasis>svn</emphasis>, <emphasis>git</emphasis>, or
           <emphasis>file</emphasis>.</para>
-		</listitem>
+        </listitem>
       </varlistentry>
 
       <varlistentry>
-		<term>repository url</term>
-		<listitem>
+        <term>repository url</term>
+        <listitem>
           <para>Remote location or folder of the files to translate.</para>
-		</listitem>
+        </listitem>
       </varlistentry>
 
       <varlistentry>
-		<term>mapping local</term>
-		<listitem>
+        <term>mapping local</term>
+        <listitem>
           <para>Name of the local folder or file, relative to the root of the
           OmegaT project.</para>
-		</listitem>
+        </listitem>
       </varlistentry>
 
       <varlistentry>
-		<term>mapping repository</term>
-		<listitem>
+        <term>mapping repository</term>
+        <listitem>
           <para>Name of the remote folder or file, relative to the repository
           url.</para>
-		</listitem>
+        </listitem>
       </varlistentry>
 
       <varlistentry>
-		<term>excludes</term>
-		<listitem>
+        <term>excludes</term>
+        <listitem>
           <para>Use wildcards (following the Apache Ant style: *, ?, **) to add
           patterns for files that should not be part of the mapping. Use a
           semicolon to separate different patterns.</para>
           <para>Example: <userinput>**/excludedfolder/**;*.txt</userinput>
           excludes files that have /excludedfolder/ in the path, as well as
           files with the <filename>.txt</filename> extension.</para>
-		</listitem>
+        </listitem>
       </varlistentry>
 
       <varlistentry>
-		<term>includes</term>
-		<listitem>
+        <term>includes</term>
+        <listitem>
           <para>As above, but for files that should be part of the mapping.
           Since files are included by default unless specifically excluded, this
           option is only necessary to specify exceptions to an exclusion
           pattern.</para>
           <para>Example: <userinput>**/*.docx</userinput> to add all .docx
           files in the project, even if they are located in an excluded
           folder.</para>
-		</listitem>
+        </listitem>
       </varlistentry>
-	</variablelist>
+    </variablelist>
   </section>
 
   <section id="how.to.setup.team.project.example.mappings">
-	<title id="how.to.setup.team.project.example.mappings.title">Example
-	mappings</title>
+    <title id="how.to.setup.team.project.example.mappings.title">Example
+    mappings</title>
 
-	<para>Default project mapping:
-	<programlisting>&lt;repository type="svn" url="https://repo_for_OmegaT_team_project"&gt;
```

Comment:
Should we change at least some of these examples to `git`, especially given that the advance project management information that will be incorporated later is entirely focused on Git.

Reply:
Please go ahead.